localizations/translated_template.json

```diff
@@ -6389,86 +6389,86 @@
     "Inpainting conditioning mask strength": "局部重绘时图像调节的蒙版屏蔽强度",
-    "Noise multiplier for img2img": "Noise multiplier for img2img",
+    "Noise multiplier for img2img": "图生图噪声倍率",
     "Apply color correction to img2img results to match original colors.": "对图生图结果应用颜色校正以匹配原始颜色",
     "With img2img, do exactly the amount of steps the slider specifies (normally you'd do less with less denoising).": "在进行图生图的时候，确切地执行滑块指定的迭代步数（正常情况下更弱的重绘幅度需要更少的迭代步数）",
-    "With img2img, fill image's transparent parts with this color.": "With img2img, fill image's transparent parts with this color.",
+    "With img2img, fill image's transparent parts with this color.": "在图生图中使用以下颜色填充透明区域",
     "Enable quantization in K samplers for sharper and cleaner results. This may change existing seeds. Requires restart to apply.": "在 K 采样器中启用量化以获得更清晰、干净的结果。这可能会改变现有的随机种子。需要保存设置并重启才能应用",
     "Emphasis: use (text) to make model pay more attention to text and [text] to make it pay less attention": "强调符：使用 (文字) 使模型更关注该文本，使用 [文字] 使其关注被减少",
     "Make K-diffusion samplers produce same images in a batch as when making a single image": "使 K-diffusion 采样器 批量生成与生成单个图像时，产出相同的图像",
     "Increase coherency by padding from the last comma within n tokens when using more than 75 tokens": "当使用超过 75 个词元(tokens)时，通过从 n 个词元中的最后一个逗号留空来提高一致性",
-    "Upcast cross attention layer to float32": "Upcast cross attention layer to float32",
+    "Upcast cross attention layer to float32": "将交叉关注层向上投射到float32",
     "Use old emphasis implementation. Can be useful to reproduce old seeds.": "使用旧的强调符实现。可用于复现旧随机种子",
-    "Use old karras scheduler sigmas (0.1 to 10).": "Use old karras scheduler sigmas (0.1 to 10).",
-    "Do not make DPM++ SDE deterministic across different batch sizes.": "Do not make DPM++ SDE deterministic across different batch sizes.",
-    "For hires fix, use width/height sliders to set final resolution rather than first pass (disables Upscale by, Resize width/height to).": "For hires fix, use width/height sliders to set final resolution rather than first pass (disables Upscale by, Resize width/height to).",
-    "Interrogate: keep models in VRAM": "反推：将模型保存在显存(VRAM)中",
-    "Interrogate: include ranks of model tags matches in results (Has no effect on caption-based interrogators).": "反推：在生成结果中包含与模型标签(tags)相匹配的等级（对基于生成自然语言描述的反推没有影响）",
-    "Interrogate: num_beams for BLIP": "反推：BLIP 的 num_beams",
-    "Interrogate: minimum description length (excluding artists, etc..)": "反推：最小描述长度（不包括艺术家等）",
-    "Interrogate: maximum description length": "反推：最大描述长度",
+    "Use old karras scheduler sigmas (0.1 to 10).": "使用旧的Karras调度器sigma值(0.1-10)",
+    "Do not make DPM++ SDE deterministic across different batch sizes.": "保留 DPM++SDE采样器 在不同的批量之间的结果差异(可以保持旧种子的复现性但改变批量时结果会发生变化)",
+    "For hires fix, use width/height sliders to set final resolution rather than first pass (disables Upscale by, Resize width/height to).": "在高分辨率修复中，使用长宽滑块设置最终分辨率(恢复旧版高分辨率修复界面，关闭放大倍率和自适应分辨率设置)",
```

Comment:
```suggestion
    "For hires fix, use width/height sliders to set final resolution rather than first pass (disables Upscale by, Resize width/height to).": "在高分辨率修复中，使用长宽滑块设置最终分辨率(恢复旧版高清修复界面，关闭放大倍率和自适应分辨率设置)",
```